tests/consensus_tests/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pytest
```

Comment:
We need to pin dependency versions so it won't break when the upstream gets updated.
Check `requirements.txt`/`requirements-freeze.txt` in other test dirs for the reference.

Also, style nit: missing trailing newline.

Reply:
This was just moved from the workflow into a `requirements` file. But I agree it is better to use a `requirements-freeze` file instead